libarchive/archive_write_set_options.3

```diff
@@ -257,6 +257,9 @@ If supported, the default value is read from
 The value is interpreted as a decimal integer specifying the
 compression level. Supported values depend on the library version,
 common values are from 1 to 22.
+.It Cm long
+Enables long distance matching. The value is interpreted as a
+decimal integer specifying log2 window size in bytes.
```

Comment:
As in the entry above, it would be good to document the range of valid values here.